tests/core/test_paths.py

```diff
@@ -42,45 +41,6 @@ def test_outdir_non_sagemaker():
         pass
 
 
-def test_tensorboard_dir_sagemaker():
```

Comment:
why removed?

Reply:
moved to pytorch test directory (https://github.com/awslabs/sagemaker-debugger/pull/150#discussion_r389860200)